src/native/pure.ts

```diff
@@ -0,0 +1,39 @@
+import { act, create, ReactTestRenderer } from 'react-test-renderer'
+
+import { TestHookProps, NativeRendererOptions, NativeRendererReturn } from '../types'
+
+import { createRenderHook, cleanup, addCleanup, removeCleanup } from '../core/index'
+
+import toRender from '../helpers/toRender'
+
+function createNativeRenderer<TProps, TResult>(
+  testHookProps: Omit<TestHookProps<TProps, TResult>, 'hookProps'>,
+  { wrapper }: NativeRendererOptions<TProps>
+): NativeRendererReturn<TProps> {
+  let container: ReactTestRenderer
+
+  const testHook = toRender(testHookProps, wrapper)
+
+  return {
+    render(props) {
+      act(() => {
+        container = create(testHook(props))
+      })
+    },
+    rerender(props) {
+      act(() => {
+        container.update(testHook(props))
+      })
+    },
+    unmount() {
+      act(() => {
+        container.unmount()
+      })
+    },
+    act
+  }
+}
+
+const renderHook = createRenderHook(createNativeRenderer)
+
+export { renderHook, act, cleanup, addCleanup, removeCleanup }
```

Comment:
It looks like all the other renderers and the auto detection export are missing `addCleanup` and `removeCleanup` in their exports (they were added after my initial POC for this).

Reply:
I can add it, i'm wondering what the use case is for exposing the addCleanup & removeCleanup API?

Reply:
[There were reasons](https://github.com/testing-library/react-hooks-testing-library/pull/432#issuecomment-687766727)... Still not sure it was the right call though.